File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Members/Fields.cs
#region Class

namespace Class
{
    public class TypeWithNoField
    {
    }

    public class TypeWithFields
    {
        Int32 Field;

        Int32 FirstField, SecondField;

        static Int32 StaticField;

        const Int32 ConstField = 1;

        readonly Int32 ReadOnlyField;

        static readonly Int32 StaticReadOnlyField;

        public Int32 PublicField;

        private Int32 PrivateField;

        protected Int32 ProtectedField;

        internal Int32 InternalField;

        protected internal Int32 ProtectedInternalField;
    }
}

#endregion

#region Struct

namespace Struct
{
    public struct TypeWithNoField
    {
    }

    public struct TypeWithFields
    {
        Int32 Field;

        Int32 FirstField, SecondField;

        static Int32 StaticField;

        const Int32 ConstField = 1;

        readonly Int32 ReadOnlyField;

        static readonly Int32 StaticReadOnlyField;

        public Int32 PublicField;

        private Int32 PrivateField;

        internal Int32 InternalField;
    }
}

#endregion

File: src/Services/Roslyn/Markify.Services.Roslyn.Fixtures/Projects/Source/Members/Fields.vb
#Region "Class"

Namespace Class
    Public Class TypeWithFields
        Dim Field As Int32

        Dim FirstField, SecondField As Int32

        Shared StaticField As Int32

        Const ConstField As Int32 = 1

        ReadOnly ReadOnlyField As Int32

        Shared ReadOnly StaticReadOnlyField As Int32

        Public PublicField As Int32

        Private PrivateField As Int32

        Protected ProtectedField As Int32

        Friend InternalField As Int32

        Protected Friend ProtectedInternal As Int32
    End Class
End Namespace

#End Region

#Region "Struct"

Namespace Struct
    Public Structure TypeWithNoField
    End Structure

    Public Structure TypeWithFields
        Dim Field As Int32

        Dim FirstField, SecondField As Int32

        Shared StaticField As Int32

        Const ConstField As Int32 = 1

        ReadOnly ReadOnlyField As Int32

        Shared ReadOnly StaticReadOnlyField As Int32

        Public PublicField As Int32

        Private PrivateField As Int32

        Friend InternalField As Int32
    End Structure
End Namespace

#End Region